index.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
     return validated;
   },
 
-  isEnabled(name) {
+  featureIsEnabled(name) {
```

Comment:
I realize this is slightly arbitrary but can you swap to `isFeatureEnabled` instead?

Reply:
Done

Reply:
Maybe forgot to push?